dev/run-pip-tests

```diff
@@ -63,10 +63,9 @@ fi
 PYSPARK_VERSION=$(python3 -c "exec(open('python/pyspark/version.py').read());print(__version__)")
 PYSPARK_DIST="$FWDIR/python/dist/pyspark-$PYSPARK_VERSION.tar.gz"
 # The pip install options we use for all the pip commands
-PIP_OPTIONS="--user --upgrade --no-cache-dir --force-reinstall "
+PIP_OPTIONS="--upgrade --no-cache-dir --force-reinstall --ignore-installed"
 # Test both regular user and edit/dev install modes.
-PIP_COMMANDS=("pip install $PIP_OPTIONS $PYSPARK_DIST"
-	      "pip install $PIP_OPTIONS -e python/")
+PIP_COMMANDS=("pip install $PIP_OPTIONS $PYSPARK_DIST")
```

Comment:
I think we don't have any documentation about editable mode for dev purpose in PySpark. I will remove this for now. I believe the installation test from the official distribute is good enough.